src/components/SingleReviewCard.jsx

```diff
@@ -0,0 +1,37 @@
+const SingleReviewCard = ({ review }) => {
+  return (
+    <div className="blog-container">
```

Comment:
There are a lot of divs here, please be mindful of the semantic tags you are using when building this app :)